docs/language/src/builtins/widgets.md

```diff
@@ -61,11 +61,14 @@ export component Example inherits Window {
 
 ## `CheckBox`
 
+A `CheckBox` should be used to selected or unselect values e.g. if the user have to choose between multiple options in a list. A `Switch` instead
+should be used to give the user the possibility to turn thing `on` or `off`.
+
```

Comment:
Nice!

A small suggestion to use the "Use XXX" phrase that I quite like for being direct and instructive:

```suggestion
Use a `CheckBox` to let the user select or deselect values, for example in a list with multiple options. Consider using a `Switch` element instead if the action resembles more something that's turned on or off.
```